### PR TITLE
Fix pylint warnings

### PR DESCRIPTION
**Pull request by mina86**

### CODE REVIEW

File: nearuplib/localnet.py [hunk 1]
```diff
@@ -57,32 +57,25 @@ def run(binary_path, home, num_nodes, num_shards, override, verbose=True, intera
...
-        with open(args_json, 'r') as config_file:
-            data = json.load(config_file)
-        data['rpc']['addr'] = f'0.0.0.0:{3030 + i}'
-        data['network']['addr'] = f'0.0.0.0:{24567 + i}'
-
-        with open(args_json, 'w') as config_file:
-            json.dump(data, config_file, indent=2)
-        i += 1
-    num_nodes = i
+        data = json.load(path.read_text())
```
Comment (marcelo-gonzalez) [inline nearuplib/localnet.py]: should be `json.loads()`?

File: nearuplib/localnet.py [hunk 2]
```diff
@@ -57,32 +57,25 @@ def run(binary_path, home, num_nodes, num_shards, override, verbose=True, intera
                    validators=num_nodes,
```
Comment (marcelo-gonzalez) [inline nearuplib/localnet.py:51]: `run_binary()` needs a string for the `home` argument, so maybe change it to `str(home)`? (I can't figure out how to attach this comment to a hunk thats not part of the PR...)

File: nearuplib/localnet.py [hunk 3]
```diff
@@ -57,32 +57,25 @@ def run(binary_path, home, num_nodes, num_shards, override, verbose=True, intera
...
+        data['rpc']['addr'] = f'0.0.0.0:{3030 + num_nodes}'
+        data['network']['addr'] = f'0.0.0.0:{24567 + num_nodes}'
+        path.write_text(json.dumps(data, indent=2))
 
     # Load public key from first node
-    with open(os.path.join(home, 'node0', 'node_key.json'),
-              'r') as node_key_file:
-        data = json.load(node_key_file)
-        public_key = data['public_key']
+    data = json.loads(open(home / 'node0' / 'node_key.json').read_text())
```
Comment (marcelo-gonzalez) [inline nearuplib/localnet.py]: It doesn't look like `read_text()` is defined for that object

File: nearuplib/localnet.py (hunk 2 above)
Comment (mina86) [reply nearuplib/localnet.py:51]: No it doesn’t? Note that subprocess.Popen accepts a list of Union[str, PathLike].

Comment (marcelo-gonzalez) [reply nearuplib/localnet.py:51]: The string requirement comes from this part:

```
    if print_command:
        print(f'Running "{" ".join(command)}"')
```

Comment (marcelo-gonzalez) [reply nearuplib/localnet.py:51]: But yea maybe makes more sense to just change that function than to change the arg to `str(home)`

**Review by marcelo-gonzalez — APPROVED** [review]

**Pull request merged**